src/cloud.js

```diff
@@ -982,7 +982,7 @@ Cloud.prototype.getUserCollections = function (
             '/collections?' +
             this.encodeDict({
                 page: page || '',
-                pageSize: page ? pageSize | 16 : '',
+                pagesize: page ? pageSize || 16 : '',
```

Comment:
haha! JSHint was complaining about this but I thought to myself "geez, Bernat surely knows something I don't" :-)